common/include/pcl/common/impl/common.hpp

```diff
@@ -70,6 +70,20 @@ pcl::getAngle3D (const Eigen::Vector3f &v1, const Eigen::Vector3f &v2, const boo
 inline void
 pcl::getMeanStd (const std::vector<float> &values, double &mean, double &stddev)
 {
+  // throw an exception when the input array is empty
+  if(values.size() == 0 )
```

Comment:
Vector has a dedicated function to test whether it is empty. Using it is less verbose and is semantically more clear:

```cpp
if (values.empty ())
```